### DISCUSSION TITLE
Asus Rog Cetra headset not recognized in GHelper

### DISCUSSION THREAD

**Discussion by Martinsicuro** [seerge/g-helper#2536] [Q&A]
Hello, 

first of all thank you for your job making this tool I will try to discover for the first time on my Asus Rog Zephyrus M16 (2022 Version)

I just found a bug, my headset, the Asus Rog Cetra with USB C cable are not recognized as Pheripherals in GHelper

They are right recognized in Armoury Crate but not in GHelper

In the screenshoot you can see I plugged my mouse (it is correctly showed) and also the Rog Cetra Headset (Not recognized)

<img width="294" alt="Ghelper" src="https://github.com/seerge/g-helper/assets/168759358/cba5ca7b-4985-47b6-83cf-4340af9258f2">

Could you help me to find a solution for this?

Thank you so much

**Comment by seerge** [comment]
@Martinsicuro hello, your device is not supported. And since I don't have your device chances that it will be added anywhere soon are minimal :)

**Comment by Martinsicuro** [comment]
Hello Seerge/G-Helper,

Could I do something to give you chances to add it in the future?

Drivers or other information about my headset?

Thank you in advance for your help

**Reply by seerge** [reply]
Headphones don't have any drivers. 

It's absolutely different type of device, that will require lots of reverse engineering in order to understand how it communicates.

What do you even set for the headphones via AC ?

**Comment by Martinsicuro** [comment]
Hello Serge,

I always set the headphones via AC because they have a lot of settings to
do in term of audio quality, bass, compressors and several other settings
and also they are still receiving a lot of updates.

This is why I asked you if they are supported in the future.

Thank you so much
*Alessandro Cerasaro*
***@***.***

**Comment by IceStormNG** [comment]
The settings you apply in AC are not applied to the headphone but to an audio filter software hat runs on your PC. 
GHelper only supports hardware if we can change settings on the device itself.

Otherwise we would have to write an audio filter, which is out of scope. There are already tools for that.
